charts/nucliadb/values.yaml

```diff
@@ -5,11 +5,16 @@ imagePullPolicy: IfNotPresent
 image: nuclia/nucliadb
 imageVersion: latest
 
+replicas: 1
```

Comment:
`replicas: 1` means there will be two instances, right?

However we are recommending having at least two replicas in the docs:
https://github.com/nuclia/docs/pull/144/files#diff-17d23fcbec1f332640a5341afcac8f13b5f20bd6a272e34bea8ebaa6af8d8a8bR33

Reply:
Agreed. I'll make the default 2!